pandas/io/tests/test_gbq.py

```diff
@@ -56,6 +56,32 @@ def _skip_if_no_private_key_contents():
         _skip_if_no_private_key_contents()
 
 
+def _check_if_can_get_correct_default_credentials():
+    try:
+        from oauth2client.client import GoogleCredentials
+        from apiclient.discovery import build
+        credentials = GoogleCredentials.get_application_default()
+        bigquery_service = build('bigquery', 'v2', credentials=credentials)
+        jobs = bigquery_service.jobs()
+        job_data = {'configuration': {'query': {'query': 'SELECT 1'}}}
+        jobs.insert(projectId=PROJECT_ID, body=job_data).execute()
+        return True
+    except:
+        return False
+
+
+def _skip_if_cant_get_correct_default_credentials():
+    if not _check_if_can_get_correct_default_credentials():
+        raise nose.SkipTest("Cannot get default_credentials "
+                            "from the environment!")
+
+
+def _skip_if_can_get_correct_default_credentials():
```

Comment:
@jreback one function is to skip a test if default credentials **can** be retrieved from the environment, while, the other function skips a test if default credentials **can't** be retrieved. Both are used in different tests. This is also the reason, I created a new function )_check_if_can_get_correct_default_credentials) after your last comment https://github.com/pydata/pandas/pull/13608#discussion_r74156084